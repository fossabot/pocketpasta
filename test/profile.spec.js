import { shallowMount } from '@vue/test-utils';
import Page from '@/pages/profile.vue';

const factory = () => shallowMount(Page, {});

describe('profile', () => {
  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
