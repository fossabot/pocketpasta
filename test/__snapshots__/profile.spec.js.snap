// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile renders properly 1`] = `"<div class=\\"container\\"><profile-stub></profile-stub></div>"`;
